builder/src/utils.ts

```diff
@@ -0,0 +1,103 @@
+// TODO: remove
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+
+/**
+ * Create the webpack ``shared`` configuration
+ */
+export function createShared({ extensions, resolutions, singletonPackages }) {
```

Comment:
I agree with the linter: a documented `IOptions` and return value would be lovely

Reply:
Yes the whole set of utils needs proper typing.